src/NotificationCenterSdk/Subscribers/Notification.cs

```diff
@@ -4,14 +4,14 @@ namespace NotificationCenterSdk.Subscribers;
 
 public class Notification
 {
-    [JsonPropertyName("id")]
+    [JsonPropertyName("_id")]
```

Comment:
I'm not sure but please let me know which endpoints required id to be provided int _id format

Reply:
the subscriber endpoints that returns an array of Notification; where the Notification object has id in that format.